### ISSUE TITLE
Implement a Progress bar while creating VM in VS Code extension

### ISSUE THREAD

**Issue by Ashchum007** [Parallels/parallels-vscode-extension#109]
**Is your feature request related to a problem? Please describe.**
With the current process, there is just an indeterminate progress indicator at the bottom of the VS Code but no specific indication of how long it will take.

**Describe the solution you'd like**
We can add the output of packer, that will show if it is downloading, or waiting for the ssh to come on, and so on.

**Additional context**
Indefinite timers for stages might at least give some indication.

**Comment by cjlapao** [comment]
@Ashchum007 Yes, we had already thought of something similar. While the waiting for ssh can be a lengthy process, that still is better than just having those progress bar running. 

what we also had in mind was to divide the 4 different stages of packer and adding a progress bar that also moved with the stages.

Not perfect but it would give a better view where the VM creation is
